include/data_management/data/soa_numeric_table.h

```diff
@@ -404,6 +431,81 @@ class DAAL_EXPORT SOANumericTable : public NumericTable
     }
 
 private:
+    services::Status generatesOffsets()
+    {
+        if (isHomogeneous())
+        {
+            if (isAllComleted())
+                DAAL_CHECK_STATUS_VAR(searchMinPointer());
+        }
+
+        return services::Status();
+    }
+
+    bool isAllComleted()
+    {
+        size_t ncols = getNumberOfColumns();
+
+        for (size_t i = 0; i < ncols; i++)
```

Comment:
Use prefix increment instead. It's also applicable for other lines in your commits